pkgs/desktops/cinnamon/cinnamon-screensaver/default.nix

```diff
@@ -82,9 +84,7 @@ stdenv.mkDerivation rec {
   NIX_CFLAGS_COMPILE = "-I${glib.dev}/include/gio-unix-2.0"; # TODO: https://github.com/NixOS/nixpkgs/issues/36468
```

Comment:
if we're using meson now, you should change this to being a mesonFlag (example is in the linked issue)